pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1663,7 +1738,10 @@ init_context(event_context_t *context, struct task_struct *task, u32 options)
     context->task.pid_id = get_task_pid_ns_id(task);
     context->task.uid = bpf_get_current_uid_gid();
     context->task.flags = 0;
-    bpf_get_current_comm(&context->task.comm, sizeof(context->task.comm));
+    ret = bpf_get_current_comm(&context->task.comm, sizeof(context->task.comm));
+    if (ret < 0)
+        tracee_error(ctx, options, BPF_ERR_GET_CURRENT_COMM, ret);
```

Comment:
Should we fail here?

Reply:
Agree; then failing `init_event_data()` in sequence.